Source/BottomView/ImageStack.swift

```diff
@@ -3,9 +3,9 @@ import UIKit
 public class ImageStack {
 
   public struct Notifications {
-    public static let imageDidPush = "imageDidPush:"
```

Comment:
It's a notification name, not a method. The name of the notification is imageDidPush, not imageDidPush:.